test/suites/pki.sh

```diff
@@ -106,6 +107,58 @@ test_pki() {
     lxc_remote remote remove pki-lxd
```

Comment:
I'd be interested in testing if we can re-add that `unrestricted` cert now that it's revoked and in the CRL.

Reply:
Done.